packages/website/src/examples/LastUpdated/LastUpdatedSpecifiedTime.example.tsx

```diff
@@ -0,0 +1,9 @@
+import {LastUpdated} from '@coveord/plasma-react';
+import * as React from 'react';
+
+const ONE_HOUR = 60 * 60 * 1000;
+
+export default () => {
+    const OneHourAgo = new Date(new Date().getTime() - ONE_HOUR);
+    return <LastUpdated time={OneHourAgo} label="Dernière modification à" />;
```

Comment:
Hoo French, is it normal?

Reply:
Well it's just an example to show that you can customize the label